types/google-apps-script/apis/analytics_v3.d.ts

```diff
@@ -1095,7 +1105,7 @@ declare namespace GoogleAppsScript {
       }
       interface UnsampledReportCloudStorageDownloadDetails {
         bucketId?: string | undefined;
-        objectId?: string | undefined;
+        anyId?: string | undefined;
```

Comment:
is `anyId` exists?

Google Apps Script Editor does not say `anyId` exists.
<img width="650" alt="image" src="https://user-images.githubusercontent.com/1213991/150136752-4f83af4b-00d6-43ce-868d-68a493242a4f.png">

In Addition, it is not Google Apps Script, Google's Dart project says `objectId` exists, instead of `anyId`.
https://pub.dev/documentation/googleapis/latest/analytics.v3/UnsampledReportCloudStorageDownloadDetails-class.html
Google Go API is also: https://pkg.go.dev/google.golang.org/api/analytics/v3#UnsampledReportCloudStorageDownloadDetails

Reply:
My mistake that is a find & replace typo from when I was replacing the `object` types with `any`

Reply:
```suggestion
        objectId?: string | undefined;
```